src/hooks/useNotifications.ts

```diff
@@ -20,4 +20,4 @@ function useFeedClient(
   }, [knock, feedId, options.source, options.tenant]);
 }
 
-export default useFeedClient;
+export default useNotifications;
```

Comment:
Is this a breaking change? And does it matter since we're still in major version 0?